app/api/auth.py

```diff
@@ -278,19 +279,19 @@ def login_user(provider):
 def verify_email():
     try:
         token = base64.b64decode(request.json['data']['token'])
-    except base64.binascii.Error:
-        raise BadRequestError({'source': ''}, 'Invalid Token')
+    except base64.binascii.Error as e:
+        raise BadRequestError({'source': e}, 'Invalid Token')
     s = get_serializer()
 
     try:
         data = s.loads(token)
-    except Exception:
-        raise BadRequestError({'source': ''}, 'Invalid Token')
+    except Exception as e:
+        raise BadRequestError({'source': e}, 'Invalid Token')
 
     try:
         user = User.query.filter_by(email=data[0]).one()
-    except Exception:
-        raise BadRequestError({'source': ''}, 'Invalid Token')
+    except Exception as e:
+        raise BadRequestError({'source': e}, 'Invalid Token')
```

Comment:
token

Reply:
Should I remove all `'source': e` to actual error message like token and all!!

Reply:
Just invalid token would be fine